.github/workflows/deploy-terraform.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: deploy-terraform
```

Comment:
For an example of a successful run of this workflow, [see here](https://github.com/ccao-data/model-sales-val/actions/runs/6386066144/job/17332084284).